src/Analysis/GWASV2/Components/DismissibleMessage/DismissibleMessage.jsx

```diff
@@ -7,8 +7,13 @@ const DismissibleMessage = ({
   title = 'Placeholder Title',
   description = 'placeholder description',
   messageType = 'success',
+  dismissMessage = null,
 }) => {
   const [open, setOpen] = useState(true);
+  const close = () => {
+    if (dismissMessage) dismissMessage();
```

Comment:
Personal preference: I recommend always including brackets with if statements. Sometimes errors can arise when other developers add statements after the if statement without them being surrounded by brackets, leading to unexpected errors and execution. 

For example these two conditionals execute foobar() under different circumstances
``` 
if (dismissMessage) 
          dismissMessage();
          foobar();
```
          
          vs
          
```
 if (dismissMessage) { 
          dismissMessage();
          foobar();
       }   
```

Reply:
I'm suspecting this was done by the linter...let me check

Reply:
Done. Using the format you proposed.